ui/raidboss/data/07-dt/trial/valigarmanda.txt

```diff
@@ -0,0 +1,251 @@
+### Worquor Lar Dor (Normal)
+# ZoneId: 1195
+
+# -p 95C2:508.4 (initial storm->ice) or -p 8D41:1014.4 (initial ice->storm)
+# -p 9691:1595.8 (adds)
+# -p 9565:1800 (end of adds)
+# -p 95C2:1872.5 (loop storm->ice)
+# -p 8D41:1872.5 (loop ice->storm)
+# -ii 8D3C 8D3E 8D40 8D59 95C2 8D50 8E09 8D4D 8D4E 8D47 8D5A 8D41 9961 87A2 6854 8D62 8D63 
+# -ii 9566 9567 8D65 8D5E 8D45
+# -ic "Feather of Ruin"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+10.3 "--sync--" StartsUsing { id: "8D3F", source: "Valigarmanda" }
+16.7 "Strangling Coil" Ability { id: "8D3F", source: "Valigarmanda" }
+31.9 "Slithering Strike" Ability { id: "8D3D", source: "Valigarmanda" }
+47.2 "Slithering Strike/Strangling Coil" Ability { id: "8D3D|8D3F", source: "Valigarmanda" }
+61.5 "Susurrant Breath" Ability { id: "8D3B", source: "Valigarmanda" }
+75.8 "Strangling Coil/Slithering Strike/Susurrant Breath" Ability { id: "8D3B|8D3D|8D3F", source: "Valigarmanda" }
+84.5 "Ice Talon" Ability { id: "8D58", source: "Valigarmanda" }
+101.5 "Strangling Coil/Slithering Strike/Susurrant Breath" Ability { id: "8D3B|8D3D|8D3F", source: "Valigarmanda" }
```

Comment:
Hmm, is there a specific reason to format multiple IDs as regex alternatives rather than an array, as `id: ["8D3D", "8D3F"]`? Obviously there's no functional difference, but typically I try to keep the formatting for netRegexes the same between trigger files and timelines (apart from single vs double quotes, of course).

Reply:
I think that's just old habits - there's really no reason it can't (and shouldn't be) expanded.  Fixed!